processor/src/errors.rs

```diff
@@ -174,6 +186,107 @@ pub enum Ext2InttError {
     UninitializedMemoryAddress(u32),
 }
 
+// ERROR CONTEXT
+// ===============================================================================================
+
+#[derive(Debug)]
+pub struct ErrorContext<'a, N: MastNodeExt>(Option<ErrorContextImpl<'a, N>>);
```

Comment:
I would add doc comments for the struct and for its public methods.

Reply:
Ah yes - done